src/azure-cli/azure/cli/command_modules/util/custom.py

```diff
@@ -174,7 +174,8 @@ def upgrade_version(cmd, update_all=None, yes=None):  # pylint: disable=too-many
 
 def demo_style(cmd, theme=None):  # pylint: disable=unused-argument
     from azure.cli.core.style import Style, print_styled_text, format_styled_text
-    format_styled_text.theme = theme
+    if theme:
+        format_styled_text.theme = theme
```

Comment:
Out of curiosity, may I ask in what scenarios will the impact of the new code be different from the old code?

Reply:
`az demo style` supports a `--theme` parameter to override the default theme. It can be `dark`, `light`, `none` (in `str`). If `bool(theme)` evaluates to `True`, it takes effect.

Reply:
OK